templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -501,4 +507,94 @@ Resources:
 {% endfor %}
 {% endif %}
 
+{% endfor %}
+
+  # Autoscaling instances
+
+{% for layer in ['Batch', 'Web', 'Solr'] %}
+  {{ layer }}LaunchTemplate:
+    Type: AWS::EC2::LaunchTemplate
+    Properties:
+      LaunchTemplateName: !Sub "${Environment}-${ApplicationName}-{{ layer|lower }}-LaunchTemplate"
+      LaunchTemplateData:
+        BlockDeviceMappings:
+          - DeviceName: "/dev/xvda"
+            Ebs:
+              DeleteOnTermination: true
+              VolumeSize: 100
+              VolumeType: "gp2"
+          - DeviceName: "/dev/sdi"
+            Ebs:
+              DeleteOnTermination: true
+              VolumeSize: 32
+              VolumeType: "gp2"
+        IamInstanceProfile:
+          Arn:
+            Fn::ImportValue: !Sub "${Environment}${ApplicationName}WebInstanceProfile"
+        ImageId: ami-03b836d87d294e89e
+        InstanceType: !Ref {{ layer }}EC2Size
+        KeyName: !Ref DefaultEC2Key
+        SecurityGroupIds:
+          - Fn::ImportValue: !Sub "${Environment}CKANManagementSG"
+          - Fn::ImportValue: !Sub "${Environment}CKANAppAsgSG"
+        UserData:
+          Fn::Base64:
+            Fn::Sub: |
+              #!/bin/sh
+              if ! (grep '/mnt/local_data' /etc/fstab >/dev/null); then
+                mkdir /mnt/local_data
+                mkfs -t xfs /dev/sdi
+                echo '/dev/sdi /mnt/local_data xfs defaults,nofail 0 2' >> /etc/fstab
```

Comment:
can't we move back to only using the primary disk at 100gb now?

Reply:
Having a separate disk gives us an extra layer of protection; filling the secondary disk doesn't crash the whole server in the way that filling the root disk would do.

Reply:
does mean needing to monitor both. also with the move to ASG, the instances will be rotated more often than disk getting filled.